journald/journald.cpp

```diff
@@ -29,7 +32,8 @@ class JournaldLoggerProcess : public Process<JournaldLoggerProcess>
 {
 public:
   JournaldLoggerProcess(const Flags& _flags)
-    : flags(_flags)
+    : ProcessBase(process::ID::generate("journald-logger")),
```

Comment:
Composing `ContainerLoggers` would be clunky.  The end result of composition would be twice the number of logger subprocesses (i.e. 2 logrotate loggers, 2 journald loggers) and a bit of funky action in a `ComposingContainerLogger`, which would need to load modules (as a module itself).